src/generators/AbstractFileGenerator.ts

```diff
@@ -7,7 +7,7 @@ export type FileGenerator = (content: string, toFile: string) => Promise<void>;
  */
 export interface AbstractFileGenerator<RenderCompleteModelOptions> {
   generateToFiles(
-    input: Record<string, unknown> | InputMetaModel,
+    input: any | InputMetaModel,
```

Comment:
Specifying the type to any kills the idea of typing. Maybe we should add another type instead?

Reply:
I mean we can, but it would be so generic, that even any makes it easier to read.

So to me `any` is probably the best way at the moment.